src/AngularPublic.js

```diff
@@ -152,7 +152,9 @@ function publishExternalAPI(angular) {
     'getTestability': getTestability,
     '$$minErr': minErr,
     '$$csp': csp,
-    'reloadWithDebugInfo': reloadWithDebugInfo
+    'reloadWithDebugInfo': reloadWithDebugInfo,
```

Comment:
This is not a just a refactoring, this exposes these two functions on the angular global, which is something we don't do lightly